packages/sui-mono/bin/sui-mono-phoenix.js

```diff
@@ -91,7 +90,15 @@ const installPackages = ({cwd = undefined, stdin = 'inherit'} = {}) => {
   const executionParams = {cwd, stdin}
   if (ci) {
     const [npm] = NPM_CMD
-    return execute([npm, ['install', ...CI_FLAGS]], executionParams)
+    return execute(
+      [npm, ['install', ...CI_FLAGS, '--only=prod']],
+      executionParams
+    ).then(() => {
+      return execute(
+        [npm, ['install', ...CI_FLAGS, '--only=dev']],
+        executionParams
+      )
+    })
```

Comment:
We forced to install Pro and Dev in every package

Reply:
¿No podemos instalar todos los paquetes de una y ya? Al quitar el `production` no debería instalarlos todos?

Reply:
Ese fue mi primer intento. Pero seguí fallando. puede ser que alguien en algún punto de la ejecución pone el NODE_ENV a producción y a efectos prácticos es como ponerle el flag de producction al comando